articles/rules/current/index.md

```diff
@@ -44,6 +44,10 @@ A Rule is a function with the following arguments:
 
 * `callback`: a function to send back potentially modified tokens back to Auth0, or an error. Because of the async nature of Node.js, it is important to always call the `callback` function, or else the script will timeout.
 
+::: note
```

Comment:
@jerriep I would move this right after the `user` bullet. We should also add this info in this doc https://auth0-docs-content-pr-4795.herokuapp.com/docs/user-profile/user-profile-structure Wdyt?